### DISCUSSION TITLE
Track object inside bounding box

### DISCUSSION THREAD

**Discussion by nilmarpublio** [ultralytics/yolov5#6055] [Q&A] [answered]
I need to count how many wheels can be detected on each car or truck that is inside a bound box in a video frame. 
So, Eeah cars and trucks are inside in bound Boxes with labels(cars or trucks, and id number) 
All detected wheels belong to the car are inside this box too with it´s labels(wheel and id number)  
Ex:    cars:  20, wheels 40
       trucks:  5, wheels 16
I´m using python, opencv and yolov5.

Someone can give me some directions?

Thanks in advance

**Comment by glenn-jocher** [comment, marked as answer]
@nilmarpublio not sure exactly what you are asking, but if your model already detects all the classes you need it's trivial to determine if one detection is contained within another.

**Reply by nilmarpublio** [reply]
My real need is to count how many cars, trucks and tires pass a certain point on the road.

**Reply by nilmarpublio** [reply]
I´m failing to do this. Have you one example ?

**Reply by glenn-jocher** [reply]
@nilmarpublio oh I see. You need a tracker like deepsort. See https://github.com/mikel-brostrom/Yolov5_DeepSort_Pytorch
